common/djangoapps/terrain/stubs/tests/test_lti_stub.py

```diff
@@ -1,10 +1,16 @@
 """
 Unit tests for stub LTI implementation.
 """
-from mock import Mock, patch
+from __future__ import absolute_import
+
 import unittest
-import urllib2
+
 import requests
+import six.moves.urllib.error  # pylint: disable=import-error
+import six.moves.urllib.parse  # pylint: disable=import-error
```

Comment:
These two imports seem unused, `six` just couldn't figure out which parts of `urllib2` were actually being used.